db/README.md

```diff
@@ -0,0 +1,28 @@
+# Database schema
+
+This folder contains the SQL code defining our database schemas. We use `sqitch` to deploy changes, and `pgTap` to test the resulting schemas.
+
+You can find the relevant team agreements and recommendations below
+
+## Sqitch changes
+
+To identify which changes are released in production environments and therefore are immutable, we rely on sqitch tags: changes that are above a tag (a line starting with `@`) in the `sqitch.plan` file should be considered immutable.
+
+When adding a sqitch change to create an object, we follow the `<object_type>/<object_name>` naming convention, where `<object_type>` is one of the following:
+
+- `schemas`: we shouldn't need to add more than the public and private schemas
+- `tables`: all tables go in there
+- `types`: all composite or enum types go in there.
+- `functions`: for functions that would result in a [postgraphile custom query](https://www.graphile.org/postgraphile/custom-queries/)
+- `computed_columns`: for functions that would result in a [postgraphile computed column](https://www.graphile.org/postgraphile/computed-columns/)
+- `mutations`: for [custom mutations](https://www.graphile.org/postgraphile/custom-mutations/)
+- `trigger_functions`: for any function returning a trigger
+- `util_functions`: for any function used primarily to increase code reuse, either in the public or private schema
+- `views`: all views go in there
```

Comment:
I really like the pattern of splitting out functions based on how they're going to be used.